packages/docs/docs/understanding-conversions/index.md

```
@@ -74,6 +74,61 @@ For example, `CLAUDE.md` converts into a Cursor Rule with `alwaysApply: true`. B
+
+`packages/foo/src/.cursor/rules/CLAUDE.mdc` with `alwaysApply: true`.
+
+This keeps the rule as "a global rule for this package, *in this package*," which is closer to the original rule's intent.
+
+---
+
+It's not always posisble to do this, though! In the above example the translation is invertible: translating that Cursor rule back to Claude will produce a `CLAUDE.md` in the original directory.
+
+But what if Cursor specifies *two* `alwaysApply: true` rules in a subdirectory?
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)